exchanges/huobi/huobi_test.go

```diff
@@ -1810,6 +1810,8 @@ var expiryWindows = map[string]uint{
 	"NQ": 282,
 }
 
+// TestPairFromContractExpiryCode ensures at least some contract codes are available and loaded with sane dates
+// Expectations are relaxed because dates are unpredictable and codes disappear intermitently
```

Comment:
All the fancy AI didn't pick up on this:
```suggestion
// Expectations are relaxed because dates are unpredictable and codes disappear intermittently
```